### ISSUE TITLE
Autosubmit GUI: Show retries in the Tree View

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#578]
In GitLab by @mcastril on Aug 20, 2020, 13:09

This is independent from showing information for full experiment new runs (after a new create), which I think should be better handled with a dropdown box that entirely sets new Views for the new run.

This is more about showing job retries in the same run. Jobs fail sometimes, specially SIMs, and it would be very useful to see if there were such retries in the Tree View.

The utility is two-fold:

* For jobs failing several times until they finally run succesfully, the GUI currently only shows information for the succesful run. Information about total queuing time and running time lost is only accessible by STATs and general metrics (by the way, are these times taken into account to do the average?).

* For jobs running well but resubmitted because a bad handling of the completion by Autosubmit, you lost the information for the "good run", which is not good. (These are maybe difficult to show as "failures" because there may actually be a COMPLETED file, but it's ok if all of them are shown as COMPLETED. It may be also that logs are only stored remotely, but Autosubmit 3.13 should better care about this).

For example, in https://earth.bsc.es/gitlab/es/auto-ecearth3/-/issues/1379 we are checking the model stability, and for a chunk that run fine at once, but then was re-run, the Tree View shows only shows the time of the re-try. https://earth.bsc.es/gitlab/es/auto-ecearth3/-/issues/1379#note_94863 . The time is only 7 seconds because we have an internal check that prevents to run SIMs using an incorrect script (the runtime was prepared to run the 3rd chunk and the Auto-Model tried to re-run the 2nd).

I think this would be difficult to implement in the Graph view, but do you think it would be possible to just add the retries to the Tree View? You can maybe add a button to Hide/Show failed jobs from it...

**Comment by kinow** [comment]
In GitLab by @wuruchi on Oct 5, 2020, 18:44

Hello @mcastril 

As part of the development of this feature, the historical job information visualization tool has been implemented in Autosubmit GUI. However, it will only work with experiments using the latest Autosubmit version and DB version 12.

![image](https://earth.bsc.es/gitlab/es/autosubmit/uploads/7ae8a5508c14fb7ee6e96634a0d505e8/image.png)

![image](https://earth.bsc.es/gitlab/es/autosubmit/uploads/1e8dc430147b48b3d6310adc67dd5e0c/image.png)

**Comment by kinow** [comment]
In GitLab by @wuruchi on Oct 9, 2020, 14:32

Hello @mcastril 

The databases still need some cleaning, but we will do that once we see that the latest version is retrieving good enough information. 

Try to load the Tree View again, it should not show that error. It happens when dealing with big experiments, probably because the .pkl is being processed.

**Comment by kinow** [comment]
In GitLab by @mcastril on Oct 15, 2020, 14:09

Thanks @wuruchi I checked it with this experiment, and previously with other ones and it is very impressive. Good work!

One question: I guess they are retries for the same JOB, no matter the experiment iteration right?

Having this, the next milestone should be dividing the experiment in iterations (separated by a `create` or `massive` setstatus) and:
* Having some separation in the Historical data view of the particular job. I think it's ok to show all the history but have some way to distinguish runs from different experiment runs.
* Have a way to reload the tree and graph and see the last runs for a particular iteration. This is what we commented about adding a combo/select box to select the iteration. I'm afraid this is more ambitious so this feature can wait.
